src/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,76 @@
+import clsx from 'clsx';
+import React from 'react';
+import styles from './Tag.module.css';
+import Text from '../Text';
+
+export const VARIANTS = [
+  'default',
+  'go',
+  'yield',
+  'stop',
+  'warning',
+  'brand',
+] as const;
+
+export type Variant = typeof VARIANTS[number];
+
+type Props = {
+  /**
+   * The color variant of the tag. New variants should be supported in the flow type,
+   * in the VARIANTS array, and by its separate style in CSS file.
+   */
+  variant?: Variant;
+  /**
+   * CSS class names that can be appended to the component for styling.
+   */
+  className?: string;
+  /**
+   * The tag icon (shouldn't provide color or size since those are determined
+   * by the color prop).
+   */
+  icon?: React.ReactNode;
+  /**
+   * The text contents of the tag, nested inside the component, in addition to the icon.
+   */
+  text?: React.ReactNode;
+  /**
+   * Adds an outline for the tag. Defaulted to no outline.
+   */
+  hasOutline?: boolean;
+};
+
+/**
+ * ```ts
+ * import {Tag} from "@chanzuckerberg/eds-components";
+ * ```
+ *
+ * This component provides a tag (pill shaped badge) wrapper.
+ */
+export const Tag = ({
+  variant = 'default',
+  className,
+  icon,
+  text,
+  hasOutline = false,
+}: Props) => {
+  return (
+    <Text
+      as="span"
+      className={clsx(
+        className,
+        styles['tag'],
+        styles[`tag--${variant}`],
+        hasOutline && styles['tag--outline'],
+      )}
+      size="sm"
+      weight="bold"
+    >
+      {icon}
+      {/* No width space to ensure height of contents */}
+      {'\u200B'}
```

Comment:
Can you double-check that we still need this? I think we also had it in the `Button` but I removed it (because it was causing issues with tests finding the component for some reason but also it didn't seem to be having an effect).

Reply:
Yah will do

Reply:
Made no difference for me as well, removing